kbs/src/attestation/keylime.rs

```diff
@@ -0,0 +1,428 @@
+            RawValue::Integer(
+                report
+                    .policy
+                    .abi_minor()
+                    .try_into()
+                    .context("unable to convert SEV-SNP report policy ABI major to int")?,
+            ),
+        ));
+        map.push((
+            RawValue::String("policy_smt_allowed".to_string()),
+            RawValue::Integer(report.policy.smt_allowed().into()),
+        ));
+        map.push((
+            RawValue::String("policy_migrate_ma".to_string()),
+            RawValue::Integer(report.policy.migrate_ma_allowed().into()),
+        ));
+        map.push((
+            RawValue::String("policy_debug_allowed".to_string()),
+            RawValue::Integer(report.policy.debug_allowed().into()),
+        ));
+        map.push((
+            RawValue::String("policy_single_socket".to_string()),
+            RawValue::Integer(report.policy.single_socket_required().into()),
+        ));
+
+        map.push((
+            RawValue::String("reported_tcb_bootloader".to_string()),
+            RawValue::Integer(report.reported_tcb.bootloader.into()),
+        ));
+        map.push((
+            RawValue::String("reported_tcb_tee".to_string()),
+            RawValue::Integer(report.reported_tcb.tee.into()),
+        ));
+        map.push((
+            RawValue::String("reported_tcb_snp".to_string()),
+            RawValue::Integer(report.reported_tcb.snp.into()),
+        ));
+        map.push((
+            RawValue::String("reported_tcb_microcode".to_string()),
+            RawValue::Integer(report.reported_tcb.microcode.into()),
+        ));
+
+        map.push((
+            RawValue::String("platform_tsme_enabled".to_string()),
+            RawValue::Integer(report.plat_info.tsme_enabled().into()),
+        ));
+        map.push((
+            RawValue::String("platform_smt_enabled".to_string()),
+            RawValue::Integer(report.plat_info.smt_enabled().into()),
+        ));
+
+        map.push((
+            RawValue::String("measurement".to_string()),
+            RawValue::Bytes(Bytes::from(report.measurement.as_slice())),
+        ));
+        map.push((
+            RawValue::String("report_data".to_string()),
+            RawValue::Bytes(Bytes::from(report.report_data.as_slice())),
+        ));
+        map.push((
+            RawValue::String("init_data".to_string()),
+            RawValue::Bytes(Bytes::from(report.host_data.as_slice())),
+        ));
+
+        return Ok(RawValue::Map(map));
+    }
+
+    Err(anyhow!("unsupported TEE"))
+}
+
+// TODO: Gather policy (used for specified claims) used to attest the evidecne and add it to the
+// Appraisal. Build the submods and add them to the EAR token.
+fn submods(_claims: RawValue) -> Result<BTreeMap<String, Appraisal>> {
+    let _map: BTreeMap<String, Appraisal> = BTreeMap::new();
+    let _appraisal = Appraisal::new();
+
+    Ok(BTreeMap::from([("".to_string(), Appraisal::new())]))
```

Comment:
This doesn't seem right.

Aside from the claims getting lost here, I wonder if you have enough information to construct an Appraisal. For instance do you plan on having an AR4SI vector in here?